lux/vislib/matplotlib/MatplotlibRenderer.py

```diff
@@ -81,6 +82,8 @@ def create_vis(self, vis, standalone=True):
             chart = LineChart(vis, fig, ax)
         elif vis.mark == "heatmap":
             chart = Heatmap(vis, fig, ax)
+        elif vis.mark == "geographical":
+            return AltairRenderer().create_vis(vis, False)
```

Comment:
@caitlynachen I noticed you mentioned in #326 that we should avoid mixing Matplotlib and Altair, except this PR warrants that sort of use case. Do you see any issues I should look out for?

Reply:
ahh that was mostly taking out useless code in Matplotlib that didn't effect any of the Matplotlib rendering. In this case I think its fine to have Altair code if there's no way to render via Matplotlib.